code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -563,6 +583,227 @@
 		new /obj/item/circular_saw(src)
 		new /obj/item/surgicaldrill(src)
 
+#define MAX_VALUE 7
+#define MIN_VALUE 6
```

Comment:
I know you're undef'ing this but could you name them to be more recognizable?
i.e `NANNY_MAX_VALUE`

Reply:
Sure